### DISCUSSION TITLE
Lost access to account with "owner" role due to Jellyfin account being deleted - how to transfer ownership to another account?

### DISCUSSION THREAD

**Discussion by ThijmenDam** [seerr-team/seerr#1038] [General]
Hi. When setting up my Jellyseerr instance, I assigned the role "owner" to an account that was imported from Jellyfin. I later deleted the Jellyfin account without realizing that it was still tied to the Jellyseerr account...

How can I transfer ownership to another Jellyseerr account?

**Comment by fallenbagel** [comment]
You'll need to edit the db to give owner permission to the new account. I would recommend just copying the new accounts deets into ID=1 (owner account) except for the permissions, leave that be

**Reply by ThijmenDam** [reply]
Great suggestion, thank you. Will do, and report back.

**Reply by ThijmenDam** [reply]
Bit late, but it worked perfectly. Thanks :-)

**Comment by AlphaSierraHotel** [comment]
Can the owner be a local user?  I'm trying to connect my Jellyseerr to a new Jellyfin install and retire the old Jellyfin install.  My owner account is tied to the old install and when I changed my Jellyfin server details in the UI, the next time I logged in, a new account was created in the db and I  couldn't authenticate as the owner account any longer.  If I could use a local Jellyseerr account as the owner, it would simplify things for me.

**Reply by fallenbagel** [reply]
Two things. Your owner IS a local account as well but if you didnt set a password beforehand you cant use local login.

AND, you can update your admin account as well. Please read our docs
https://docs.jellyseerr.dev/troubleshooting#account-does-not-have-admin-privileges

**Reply by AlphaSierraHotel** [reply]
Not sure how I missed that in the docs.  Thanks of pointing it out.
